estate/models/estate_property.py

```diff
@@ -0,0 +1,110 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real estate property"
+
+    name = fields.Char(
+        string="Title",
+        required=True,
+        translate=True,
+    )
+    description = fields.Text(
+        string="Description",
+        required=False,
+        translate=True,
+        copy=False,
+    )
+    image = fields.Image(
+        "Image",
+        copy=False,
```

Comment:
I find it strange that you use so much the `copy=False`, so I'll just explain it quickly and then you can decide whether that's what you want or not.

`copy=False` makes it so that when you duplicate a record of the model, the fields with `copy=False`'s values will not be set in the new duplicated record. Usually we use `copy=False` on fields that limitate the possibilities on a record so that the new record can be modified as much as desired before validation. For example, we don't copy the state of an invoice when we duplicate it beause if it is validated, then you can't edit the customer, the date, the products, etc on the new record. But if you put too many `copy=False` then it just acts as the `New` button because it will create an empty record. So you want to use as little `copy=False` as possible to have the closest copy of the duplicated record as possible while also maintaining manoeuvrability.

In this case, I don't think `image` should have `copy=False` for example but maybe you meant to do it so I'll let you decide which fields really needs to be `copy=False` and we'll probably talk about it again later anyway :smile: